modules/auxiliary/scanner/http/cisco_ssl_vpn.rb

```diff
@@ -84,6 +84,7 @@ def check_conn?
     begin
       res = send_request_cgi('uri' => '/', 'method' => 'GET')
       vprint_good("Server is responsive...")
+      return true
```

Comment:
I was lazy again :-) and assumed it's ok to do it after "Server is responsive..." :-)  Thanks for raising the question.